### ISSUE TITLE
Not able to init in TimberImage with iid=//www.gravatar.com/avatar/0da2f1a9340d6bf196b510f6c107a248?s=96&#038;r=g&#038;d=mm

### ISSUE THREAD

**Issue by henrywright** [timber/timber#1748]
We are getting a message in our error log which relates to Timber.

> Not able to init in TimberImage with iid=//www.gravatar.com/avatar/0da2f1a9340d6bf196b510f6c107a248?s=96&#038;r=g&#038;d=mm

The message seems to be generated in the `init()` method in timber-library/lib/image.php

We are using WP 4.9.7, PHP 7.2 and Timber 1.7.1

**Comment by jarednova** [comment]
@henrywright I think it's recognizing that as a server path as opposed to a URL — can you see if there's a way to prefix the gravatar URL with `http` or `https`?

**Comment by henrywright** [comment]
Can a server path have a `//` in it? If not then you could just do this:
```
if ( strstr( $iid, '//' ) ) {
    $this->init_with_url( $iid );
    return;
}
```

**Comment by jarednova** [comment]
Thanks @henrywright for the issue and PR! Just merged in #1749 with @mahagr's adjustment

**Comment by henrywright** [comment]
Thanks @jarednova 

@mahagr's suggestion is a good improvement
